email_api/email.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+nodeMailer = require('nodemailer');
+var sendemail = require('./send_email');
+
+router.get('/',function(req,res,next)
+{
+  console.log("inside get request of email file");
+  next();
+})
+router.use(email_sender);
+
+function email_sender(req,res) {
+  var mail_obj = {
```

Comment:
follow camel case email_sender is not following camel case convention

Reply:
this has been resolved